### ISSUE TITLE
M12 Dataset Availability?

### ISSUE THREAD

**Issue by eyildiz-ugoe** [isl-org/ZoeDepth#73]
In the paper supplement it is mentioned in the table 8, that the relative depth backbone has been trained on a mix of 12 datasets. Yet, neither here nor anywhere else it is shared how these are loaded. I think the training script itself is also missing.

**Comment by northagain** [comment]
no, midas is trained using 12 datasets, it only used kitti and nyu
